src/demo/sections/DataTable.vue

```diff
@@ -58,37 +52,56 @@ export default defineComponent({
   components: { DataTable, Checkbox },
   data() {
     const data: {
+      selected: any[],
       sortBy: string
       sortDir: DataTableColumnSort
       columns: DataTableColumn
       rows: DataTableRow
     } = {
+      selected: [],
       sortBy: 'memberCount',
       sortDir: 'desc',
       columns: [
         {
           label: 'Name',
-          value: 'name'
+          value: 'name',
+          search: true
         },
         {
           label: 'Member Count',
-          value: 'memberCount'
+          value: 'memberCount',
+          search: true
+        },
+        {
+          label: 'Created',
+          value: 'created',
+          search: true
         },
         {
           label: 'Roles',
-          value: 'roles'
-        }
+          value: 'roles',
+          search: true
+        },
       ],
       rows: [
-        { name: 'Staging Team', memberCount: 57, roles: 'Admin' },
-        { name: 'Data Science', memberCount: 405, roles: 'Admin' },
-        { name: 'Dev Ops', memberCount: 22, roles: 'Admin' },
-        { name: 'Production Team', memberCount: 35, roles: 'Admin' },
-        { name: 'Winter Interns', memberCount: 90, roles: 'Admin' }
+        { name: 'Staging Team', memberCount: 57, roles: 'Admin', created: new Date() },
+        { name: 'Data Science', memberCount: 405, roles: 'Admin', created: new Date() },
+        { name: 'Dev Ops', memberCount: 22, roles: 'Admin', created: new Date() },
+        { name: 'Production Team', memberCount: 35, roles: 'Admin', created: new Date() },
+        { name: 'Winter Interns', memberCount: 90, roles: 'Admin', created: undefined }
```

Comment:
if you use random dates, it might help illustrate searching that column better

Reply:
Agreed - I'll make that update later though so this doesn't get blocked